codespell_lib/data/dictionary.txt

```diff
@@ -2328,6 +2328,7 @@ aformentioned->aforementioned
 afrer->after
 afterall->after all
 afterw->after
+aftewards->afterwards
```

Comment:
For consistency with other misspelling fixes, I would suggest:
```suggestion
afteward->afterward
aftewards->afterwards
 ```

Reply:
Appreciate it, @DimitriPapadopoulos!

Reply:
Could you add `afteward` to your pull request ? I cannot add it using the GitHub web interface, it cannot handle large files and prints nonsensical error messages.

Reply:
Sorry about that. I've been away from my computer. I have added that commit. This should be ready for you to re-review.